### DISCUSSION TITLE
ImportError: cannot import name QED

### DISCUSSION THREAD

**Discussion by danielmanu93** [rdkit/rdkit#4370] [Q&A] [answered]
I have rdkit installed on my server and trying to execute my code that has rdkit import "from rdkit.Chem import QED", but I get an "import error: cannot import name QED" as in shown in the attached image. Any ideas about how to get this fixed are welcomed.
![1](https://user-images.githubusercontent.com/67971334/127249457-ced2d385-0ac0-4ad0-b7a7-9831eba90bb2.png)

**Comment by greglandrum** [comment, marked as answer]
How was the RDKit installed on the server and which version is installed?

**Reply by danielmanu93** [reply]
I have tried using 3 commands: "conda install rdkit -c rdkit", "conda install -c rdkit rdkit" and "sudo apt-get install python-rdkit librdkit1 rdkit-data" but still getting the same error. The version is 2017.03.3'

**Reply by greglandrum** [reply]
That's an ancient version of the RDKit. The QED descriptor was added in v2017.09.1, but you should really try to get v2021.03.x if possible
We now recommend that you install the RDKit from conda-forge instead of the RDKit channel... those builds are no longer kept up-to-date.

**Reply by danielmanu93** [reply]
Sure, I created a conda environment and installed the 2018 version and it's working. Thanks
